test/Index/roles.swift

```diff
@@ -336,7 +336,7 @@ extension ExtendMe where Self: Whatever {}
 
 var anInstance = AClass(x: 1)
 // CHECK: [[@LINE-1]]:18 | class/Swift | AClass | s:14swift_ide_test6AClassC | Ref,RelCont | rel: 1
-// CHECK: [[@LINE-2]]:18 | constructor/Swift | init(x:) | s:14swift_ide_test6AClassC1xACSi_tcfc | Ref,Call,RelCont | rel: 1
+// CHECK: [[@LINE-2]]:18 | constructor/Swift | init(x:) | s:14swift_ide_test6AClassC1xACSi_tcfc | Ref,Call,Dyn,RelCont | rel: 1
```

Comment:
Hmm, that doesn't seem right to me, unless the `init` is marked dynamic, there should be no dynamic dispatch here.

Reply:
yea, this was the core reason we decided to drop this for now